runtime/src/main/cpp/KAssert.h

```diff
@@ -25,17 +25,17 @@
 #define STRINGIFY(x) #x
 #define TOSTRING(x) STRINGIFY(x)
 
-RUNTIME_NORETURN void RuntimeAssertFailed(const char* location, const char* message, ...);
+RUNTIME_NORETURN NO_INLINE void RuntimeAssertFailed(const char* location, const char* message, ...);
 
 namespace internal {
 
 inline RUNTIME_NORETURN void TODOImpl(const char* location) {
     RuntimeAssertFailed(location, "Unimplemented");
 }
 
-// TODO: Support format string when `RuntimeAssertFailed` supports it.
-inline RUNTIME_NORETURN void TODOImpl(const char* location, const char* message) {
-    RuntimeAssertFailed(location, message);
+template <typename ... Args>
+inline RUNTIME_NORETURN void TODOImpl(const char* location, const char* message, Args ... args) {
```

Comment:
The usual pattern for forwarding is:
```cpp
template <typename... Args>
void f(Args&&... args) {
   g(std::forward<Args>(args)...);
}
```
Note the use of `&&` in the argument list, and `std::forward` where forwarding happens. This pattern allows copying the arguments that should be copied and moving the arguments that should be moved.